torchvision/transforms/functional.py

```diff
@@ -50,6 +52,7 @@ def _interpolation_modes_from_int(i: int) -> InterpolationMode:
     InterpolationMode.NEAREST: 0,
     InterpolationMode.BILINEAR: 2,
     InterpolationMode.BICUBIC: 3,
+    InterpolationMode.NEAREST_EXACT: 0,
```

Comment:
The only instance of where we want the reverse mapping, i.e. from `int` to `InterpolationMode`, comes from the `_interpolation_modes_from_int` function above. AFAIK, this is only needed until the deprecation period for passing `int`'s is over and will be removed afterwards. Is that true?

If yes, I think it is ok to also assign `0` as the Pillow interpolation mode here as we already do for `InterpolationMode.NEAREST`. For the Pillow side this makes sense, since they only have one nearest interpolation. 

The only downside to this is that users cannot access `InterpolationMode.NEAREST_EXACT` through and `int`. But since that is deprecated anyway, I don't think this is an issue.

Reply:
I think as long as `0` is assigned back to `InterpolationMode.NEAREST` for BC, we should be OK.